packages/react-components/react-tree/src/components/TreeItem/useTreeItemStyles.styles.ts

```diff
@@ -24,61 +21,27 @@ const useRootStyles = makeStyles({
       { [treeItemLevelToken]: index + 1 },
     ]),
   ) as Record<StaticLevelProperty, GriffelStyle>),
-});
-
-/**
- * Styles for the content slot
- */
-const useContentStyles = makeStyles({
   base: {
     position: 'relative',
     cursor: 'pointer',
-    display: 'flex',
+    display: 'grid',
+    gridTemplateRows: 'auto auto',
```

Comment:
is the change from the left padding to grid area necessary for this PR?

Reply:
Unfortunately yes, notice that padding applies to the whole tree item meanwhile by using grid we're only applying the "padding" on the content, and not on the subtree